frappe/custom/doctype/customize_form_field/customize_form_field.json

```diff
@@ -80,6 +81,12 @@
    "oldfieldtype": "Data",
    "search_index": 1
   },
+  {
+    "fieldname": "module",
+    "fieldtype": "Link",
+    "label": "Module (for export)",
+    "options": "Module Def"
+   },
```

Comment:
Whenever a doctype's json file is changed, the modified value should also change (which gets automatically changed if you update the doctype and save) because of it the changes get affected on migrate